homeassistant/components/home_connect/__init__.py

```diff
@@ -139,6 +145,20 @@ def find_appliance(
     raise ValueError(f"Appliance for device id {device_entry.id} not found")
 
 
+def _get_appliance_or_rise_service_validation_error(device_id, hass):
+    """Return a Home Connect appliance instance or raise a service validation error."""
+    try:
+        return _get_appliance(hass, device_id)
+    except (ValueError, AssertionError) as err:
+        raise ServiceValidationError(
+            translation_domain=DOMAIN,
+            translation_key=SVE_TRANSLATION_KEY_APPLIANCE_NOT_FOUND,
+            translation_placeholders={
+                SVE_TRANSLATION_PLACEHOLDER_DEVICE_ID: device_id,
```

Comment:
I find this use of constants very confusing to follow.
IMO they do not add much value here, and these two are single-use.